pkg/cmd/cli/cli.go

```diff
@@ -62,36 +62,40 @@ func NewCommandCLI(name, fullName string) *cobra.Command {
 	in := os.Stdin
 	out := os.Stdout
 
+	// OpenShift-specific commands
+	cmds.AddCommand(cmd.NewCmdBuildLogs(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdCancelBuild(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdConfig(fullName, "config"))
+	cmds.AddCommand(cmd.NewCmdDeploy(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdEdit(fullName, f, out))
+	cmds.AddCommand(cmd.NewCmdEnv(fullName, f, os.Stdin, out))
 	cmds.AddCommand(cmd.NewCmdLogin(fullName, f, in, out))
 	cmds.AddCommand(cmd.NewCmdLogout("logout", fullName+" logout", fullName+" login", f, in, out))
+	cmds.AddCommand(cmd.NewCmdNewApplication(fullName, f, out))
```

Comment:
Why did you reorder these?  The order is significant, and is intentional.  Please revert the ordering.